ocf_datapipes/transform/xarray/pv/assign_daynight_status.py

```diff
@@ -0,0 +1,84 @@
+import logging
```

Comment:
Ill ignore this as this is in #126

Reply:
Yes, that's true, but as the dropping of systems depends on the `assign_daynight_status` as well, I think if the python tests `Python package tests / call-run-python-tests / run-python-tests (3.9, ubuntu-latest) (push)` will fail if #127 is not included in this. I'll push without #126 and see if it successful.

Reply:
Without the `assign_daynight_status`, the python tests in the GitHub are unsuccessful, so adding #126 in this #127 PR as well.